### ISSUE TITLE
BUG:Images with no created at metadata are not accesible 

### ISSUE THREAD

**Issue by Rishab87** [AOSSIE-Org/PictoPy#144]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

On different filesystems , like on linux filesystem where we also have to mount other drives, their created at metadata may not be present in image due to which if we change images folder in settings, images dont display on home page for that. So I think its a good idea to fallback to modified metadata if created at is not present in image.

In simple terms in code we were accessing created at metadata to display images on home page but on many file systems like in linux where we can't access created at metadata of other images which are present in other drives their due  this images are not visible on home as images on home are soley relying on created at metadata.

### Record

- [X] I agree to follow this project's Code of Conduct

**Comment by Rishab87** [comment]
@Pranav0-0Aggarwal can you please assign this, I've found its solution

**Comment by Pranav0-0Aggarwal** [comment]
done

**Comment by Rishab87** [comment]
@Pranav0-0Aggarwal I've fixed this issue, please check!
